docs/benchmarking-workflow.md

```diff
@@ -206,13 +219,15 @@ Pass the path to CoreRun using `--coreRun` argument. In both CoreCLR and CoreFX
 * "C:\Projects\coreclr\bin\tests\Windows_NT.x64.Release\Tests\Core_Root\CoreRun.exe"
 * "C:\Projects\corefx\bin\runtime\netcoreapp-Windows_NT-Release-x64\CoreRun.exe"
 
+**Note:** BenchmarkDotNet expects a path to `CoreRun` file, not to `Core_Root` folder.
```

Comment:
On unix, the name is case sensitive and the file is named `corerun`. It may be worth mentioning. E.g.:
BenchmarkDotNet expects a path to `CoreRun.exe` file (`corerun` on Unix) ...

Reply:
@janvorli fixed, thanks!